Generals/Code/GameEngineDevice/Source/W3DDevice/GameClient/Drawable/Draw/W3DLaserDraw.cpp

```diff
@@ -234,6 +234,7 @@ W3DLaserDraw::~W3DLaserDraw( void )
 	}  // end for i
 
 	delete [] m_line3D;
+	REF_PTR_RELEASE(m_texture);
```

Comment:
All other memory leak fixes have comments, so might as well add one here for consistency.